src/jrd/ini.epp

```diff
@@ -411,7 +411,7 @@ namespace
 			const char* users[] = {userName, "PUBLIC"};
 			int grantOptions[] = {WITH_GRANT_OPTION, 0};
 
-			for (unsigned i = 0; i < FB_NELEM(users); i++)
+			for (int i = 0; i < FB_NELEM(users); i++)
```

Comment:
Why ?

Reply:
`FB_NELEM` is signed. It fixes a warning.

Reply:
This PR waited more than a year. Forever is just a little longer so why should I care?..

Reply:
You get one thing wrong: it is not me who wait, but Firebird users.